composer.json

```diff
@@ -143,6 +143,7 @@
         "twig/markdown-extra": "^2.12|^3"
     },
     "conflict": {
+        "doctrine/annotations": "1.13.0",
```

Comment:
I'd be in favor of removing that line as it's going to stay here for years otherwise.

Reply:
My plan was to resolve this in 5.4 by bumping Annotations to `^1.13.1`. I really want to avoid that the changed cache warmer is run against 1.13.0.

Reply:
My concern is that maintaining those list of constraints is costly, so we'd better have a little of them as possible. Any non-latest version (in same minor) is borken by definition :)

Reply:
All right, removed!